e2e/local-e2e.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+################################################################################
+#
+# This script is meant to be run from the root of this project
+#
+################################################################################
+
+set -euo pipefail
+
+function run() {
+  minikube_up
+  generate_manifests
+  deploy
+}
+
+# Create local minikube cluster and deploys the dev env for parca and parca agent
+function minikube_up(){
+  echo "Spinning up a parca dev cluster"
+  minikube start -p parca-e2e \
+    --container-runtime=docker \
+    --insecure-registry="localhost:5000"\
+    --driver=kvm2 \
```

Comment:
Let's make this configurable. So that people on other platforms can run them by specifying it.